src/Symfony/Bundle/SecurityBundle/DependencyInjection/Compiler/AddSecurityVotersPass.php

```diff
@@ -40,6 +41,20 @@ public function process(ContainerBuilder $container)
             throw new LogicException('No security voters found. You need to tag at least one with "security.voter"');
         }
 
+        foreach ($voters as $voter) {
+            $class = $container->getDefinition($voter->__toString())->getClass();
```

Comment:
Are you sure? That's harder to trace back (I'm not sure why toString is used either way)